eng/pipelines/templates/jobs/archetype-sdk-client.yml

```diff
@@ -15,11 +15,11 @@ jobs:
           DOTNET_SKIP_FIRST_TIME_EXPERIENCE: 1
           DOTNET_CLI_TELEMETRY_OPTOUT: 1
           DOTNET_MULTILEVEL_LOOKUP: 0
-      - task: PublishPipelineArtifact@0
+      - task: PublishPipelineArtifact@1
```

Comment:
Out of curiosity why did you bump to the later PublishPipelineArtifact task?

Reply:
Just boyscouting. Typically I update to the latest supported versions of the tasks after reading the release notes, looking at the code changes, and testing the task update. In this case, `PublishPipelineArtifact` was version bumped but basically no changes were made except for renaming one parameter. The reason for the version bump was because the pipeline artifacts are moving out of beta and they don’t want to ship while telling customers to use a V0 task.